plugins/inputs/prometheus/metrics_receiver_test.go

```diff
@@ -110,3 +114,20 @@ func Test_metricAppender_Commit(t *testing.T) {
 	}
 	assert.Equal(t, expected, *pmb[0])
 }
+
+func Test_loadConfigFromFile(t *testing.T) {
+	os.Setenv("POD_NAME", "collector-1")
```

Comment:
Could you clean this env var once the test is done (good for avoiding any side effects on other tests)